src/remove-empty-keys.js

```diff
@@ -0,0 +1,28 @@
+export {
```

Comment:
Could you add `export * from './remove-empty-keys'` to [`index.js`](https://github.com/kentcdodds/webpack-config-utils/blob/master/src/index.js)?